includes/class-satispress-package-plugin.php

```diff
@@ -218,9 +218,72 @@ public function get_version() {
 	 */
 	protected function get_data( $prop ) {
 		if ( empty( $this->data ) ) {
-			$this->data = get_plugin_data( $this->get_file(), false, false );
+			$this->data = $this->get_plugin_data( $this->get_file(), false, false );
```

Comment:
This uses the new copied internal get_plugin_data function with the added "Plugin Type" option.